index.html

```diff
@@ -37,154 +36,213 @@ <h1> Hey there. I’m Alex.<span class="green" > <br> I’m a software developer
+                        <li class="language">html</li>
+                        <li class="language">css</li>
+                        <li class="language">javascript</li>
+                        <li class="language">Ruby on rails</li>
+                    </ul>
+                    <div class="butt"><button class="btn" type="button"> See Project</button></div>
+                </article>
+                 <article class="project">
+                    <div class ="image-placeholder"></div>
+                    <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
+                    <ul class ="lang">
+                        <li class="language">html</li>
+                        <li class="language">css</li>
+                        <li class="language">javascript</li>
+                        <li class="language">Ruby on rails</li>
+                    </ul>
+                    <div class="butt"><button class="btn" type="button"> See Project</button></div>
+                </article>
+                </div>
+                </div>
+            </div>
+            </section>
 
-       <section class="aboutme-background">
-       <section class="aboutme section">
-           <div class="aboutme-container">
-           <div class="about2">
-              
+            <section class="aboutme-background">
+            <section class="aboutme section">
+                <div class="aboutme-container">
+                <div class="about2">
+                    <h2>About me</h2>
 
-           <div class="about">
-            <h2>About me</h2>
-           </div>
-           <p class="about-text">
-           I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-           don’t hestiate to contact me.
-           </p>
-           <div class="butt resume-container"><button class="btn btn-resume" type="button"> Get My Resume</button></div>
-       </div>
-           </div>  
-           
-           <div class="skills-container">
-           <article class="project2">
-           <h3 class="language-icon"><img src="./images/icon-languages (1).png" alt=""></h3>
-           <h4 class="lang-text">Languages </h4>
-               <ul class="lang-text-list">
-                   <li  class="li-lang">Javascript</li>
-                   <li class="li-lang">Ruby</li>
-                   <li class="li-lang">html</li>
-                   <li class="li-lang">css</li>
-               </ul>
+                <div class="about">
+                </div>
+                <div class="about-text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+                </div>
+                <div class="butt "><button class="btn btn-resume" type="button"> Get My Resume</button></div>
```

Comment:
As suggested by the previous reviewer, this line should not be breaking, kindly restyle it so that it will be on the same lime 😎
You can try using this style...
 ```
width: 159px;
  height: 48px;
  text-align: center;
  margin: 0 auto;
  padding: 0.8em;
  margin-top: 1em;
```